nav2_system_tests/maps/map_circular.yaml

```diff
@@ -4,4 +4,5 @@ origin: [-10.000000, -10.000000, 0.000000]
 negate: 0
 occupied_thresh: 0.65
 free_thresh: 0.196
+map_type: occupancy
```

Comment:
FYI - map_type should be optional and default to "occupancy." My PR #410 fixes this (I didn't have it correct in the original map server update PR).  Doesn't hurt to have it, but it shouldn't have been necessary.